capsules/src/lpm013m126.rs

```diff
@@ -543,7 +543,17 @@ where
 
     fn set_brightness(&self, _brightness: usize) -> Result<(), ErrorCode> {
         // TODO: add LED PWM
-        Ok(())
+        let scheduled = schedule_deferred(
+            self.deferred_caller,
+            &self.command_complete_callback,
+            "set_brightness complete",
+        );
```

Comment:
Wait why does this need to set a deferred call? The client should not expect a callback if the `set_brightness()` command fails.

Reply:
You're right, I did that change too mechanically.